archive.php

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+</head>
+<body>
+	<?php
+	include 'navbar.php';
+	?>
+	<div id="container">
+		<script>
+		var youtube_videos = ['https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
+								,'https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
+							];
+		for(i=0;i<youtube_videos.length;i++){
+			$("#container").append("<div class='col-lg-4 col-md-4 col-sm-6 col-sm-12' id='vid'><iframe frameborder='0' allowfullscreen></iframe></div>");
+			
+		}
+			var video = $('#vid');
+			
+		for(i=0;i<youtube_videos.length;i++){
+			
+			video.children('iframe').attr('src',youtube_videos[i]);
+			video = video.next();
+		}
+		</script>
+	</div>
+<script src="arcive.js"></script>	
```

Comment:
Made the changes.

On Sun, Aug 30, 2015 at 1:05 AM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38269511>
> :
>
> > +								,'https://www.youtube.com/embed/15tLVw0V-bM','https://www.youtube.com/embed/15tLVw0V-bM'
> > +							];
> > +		for(i=0;i<youtube_videos.length;i++){
> > +			$("#container").append("<div class='col-lg-4 col-md-4 col-sm-6 col-sm-12' id='vid'><iframe frameborder='0' allowfullscreen></iframe></div>");
> > +			
> > +		}
> > +			var video = $('#vid');
> > +			
> > +		for(i=0;i<youtube_videos.length;i++){
> > +			
> > +			video.children('iframe').attr('src',youtube_videos[i]);
> > +			video = video.next();
> > +		}
> > +		</script>
> > +	</div>
> > +<script src="arcive.js"></script>	
>
> @hitesh4 <https://github.com/hitesh4> I don't see this file in the PR.
> Remove this import from this file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38269511>
> .
>